ironic/drivers/modules/drac/raid.py

```diff
@@ -727,6 +765,45 @@ def _filter_logical_disks(logical_disks, include_root_volume,
     return filtered_disks
 
 
+def _change_physical_disk_conversion(
+        node, mode=None, controllers_to_physical_disk_ids=None):
+    """
+    :param node: an ironic node objec
+    :param mode: enumeration that indicates the mode
+                     to change the disks to.
+    :param controllers_to_physical_disk_ids:Dictionary of controllers and
+                                            corresponding disk ids to convert
+                                            to the requested mode
+    :return: a dictionary containing:
+             - conversion_results, a dictionary that maps controller ids
+                    to the conversion results for that controller.  The
+                    conversion results are a dict that contains:
+                    - The is_commit_required key with the value always set to
+                      True indicating that a config job must be created to
+                      complete disk conversion.
+                    - The is_reboot_required key with a RebootRequired
+                      enumerated value indicating whether the server must be
+                      rebooted to complete disk conversion.
+    """
+    change_disk_state = change_physical_disk_state(
+        node,
+        mode,
+        controllers_to_physical_disk_ids=controllers_to_physical_disk_ids)
+
+    LOG.info("Change_disk_state return-- : {}".format(str(change_disk_state)))
```

Comment:
In the line above, would you mind changing "Change_disk_state" to "change_physical_disk_state"?

Reply:
Probably could remove the "-- " as well.

Reply:
And this should probably be logged at the debug level.

Reply:
Yes i have added this just for the debug level personally. I will remove this when i raise the actual patch against upstream ironic.